templates/templates.js

```diff
@@ -60,4 +60,57 @@
       ["README.md"]
     ]
   }
-}]
\ No newline at end of file
+}, {
+  "name": "Empty AssemblyScript Project",
+  "description": `# Empty AssemblyScript Project
+
+[AssemblyScript](https://github.com/AssemblyScript/assemblyscript) compiles strictly typed TypeScript to WebAssembly using Binaryen.
+
+See the [AssemblyScript wiki](https://github.com/AssemblyScript/assemblyscript/wiki) for further instructions and documentation.
+`,
+  "icon": "svg/file_type_typescript.svg",
+  "project": {
+    "name": "Project",
+    "directory": "empty_ts",
+    "children": [{
+      "name": "src",
+      "children": [{
+        "name": "main.ts",
+        "type": "typescript"
+      }, {
+        "name": "main.js",
+        "type": "javascript",
+      }, {
+        "name": "main.html",
+        "type": "html",
+      }]
+    }, {
+      "name": "build.ts",
+      "type": "typescript",
+    }, {
+      "name": "README.md",
+      "type": "markdown",
+    }],
+    "openedFiles": [
+      ["README.md"]
+    ]
+  },
+  "onload": `
```

Comment:
Templates should not be special in any way, they should more or less behave like fiddles others create with the exception that they are shown in a list when no fiddle is specified. In other words, we can't attach logic to templates that cannot be added to fiddles.